crates/oxc_prettier/src/format/mod.rs

```diff
@@ -1405,7 +1402,7 @@ impl<'a> Format<'a> for BigIntLiteral<'a> {
         let text = self.span.source_text(p.source_text);
         // Perf: avoid a memory allocation from `to_ascii_lowercase`.
         if text.contains(|c: char| c.is_lowercase()) {
```

Comment:
Same here?

Reply:
Then the formatter doesn't need cow at the moment🤣

Reply:
`if contain` condition is deleted, will the contents still be retained? 

This `fn format` seems there's nothing left.